content/docs/guides/securing-tcp.mdx

```diff
@@ -71,104 +69,123 @@ This will install a trusted CA and generate a new wildcard certificate:
 - `_wildcard.localhost.pomerium.io.pem`
 - `_wildcard.localhost.pomerium.io-key.pem`
 
-To provide your own certificates through another mechanism, please overwrite these files or update `docker-compose.yaml` accordingly.
-
 ## Configure
 
 ### Pomerium
 
-Update `config.yaml` with your IdP settings and desired policy if adopting for your environment
+Add the configuration below to your configuration file:
 
 <PomeriumConfig />
 
+:::note
+
+Make sure you update each `example.com` instance to match the domain of your email address.
+
+:::
+
 ### Docker Compose
 
-Create a `docker-compose.yaml` file to run Pomerium and, optionally, the services being demonstrated.
+Create a `docker-compose.yaml` file to run Pomerium and the services you will connect to over TCP.
 
-Included in our compose file:
+The following services are included in the Docker Compose file:
 
-- SSH
-- Postgres
-- Redis
+- **Redis**
+- **SSH**
+- **Postgres**
 
 <DockerCompose />
 
 ## Connect
 
-To connect to your service, ensure [`pomerium-cli`](/docs/deploy/clients/pomerium-cli) is in your `$PATH` and run the `tcp` command, specifying the service you wish to reach.
+To connect to your service, ensure `pomerium-cli` is in your `$PATH` and run the `tcp` command, specifying the service you wish to reach.
 
 ```bash
 pomerium-cli tcp [hostname]:[port]
 ```
 
 `pomerium-cli` will select a random port on `localhost` by default, but you can specify a port manually if desired. Keep reading for some specific application examples using the sample `docker-compose.yaml`.
 
+:::caution
+
+This guide uses the `--disable-tls-verification` flag to bypass TLS requirements. Without this flag, TCP connections will fail unless you include a trusted CA and certificate in your configuration.
```

Comment:
(optional)

We might want to say something more like "If you haven't configured a trusted certificate, you'll need to add the `--disable-tls-verification` flag to the `pomerium-cli` commands below." (and then omit this flag from the written example commands)